src/faucet.e2e.ts

```diff
@@ -11,7 +11,7 @@ const randomAddress = () => createTestKeyring().addFromSeed(randomAsU8a(32)).add
 describe("Faucet E2E", () => {
   const matrix = axios.create({ baseURL: "http://localhost:8008" });
   const webEndpoint = axios.create({ baseURL: "http://localhost:5555" });
-  const PARACHAIN_ID = 100; // From the zombienet config.
+  const PARACHAIN_ID = 1000; // From the zombienet config.
```

Comment:
So we have to change the zombienet config as well - it's [here](https://github.com/paritytech/polkadot-testnet-faucet/blob/570eb9e731bb32f9cb60e0dabf8595d3118ec57a/e2e/zombienet.native.toml#L17).

Reply:
Is this config from zombie net valid, or was it set only for the e2e tests?

Reply:
IT PASSED!